app/Model/SubtaskTaskConversionModel.php

```diff
@@ -39,7 +39,7 @@ public function convertToTask($project_id, $subtask_id)
         ));
 
         if ($task_id !== false) {
-            $this->taskLinkModel->create($task_id, $subtask['task_id'], 6);
+            $this->taskLinkModel->create($task_id, $subtask['task_id'], 7);
```

Comment:
The person that did this change hardcoded the link ID. See this commit https://github.com/kanboard/kanboard/commit/d4749a44025b1518cd8f83941f960e2f699b69b7

Your change will work for MySQL, but it's going to break for Sqlite and Postgres users.

The right way of fixing this is to fetch the link ID by using the name. There are few methods available in LinkModel class: https://github.com/kanboard/kanboard/blob/master/app/Model/LinkModel.php

Reply:
> The right way of fixing this is to fetch the link ID by using the name. There are few methods available in LinkModel class: https://github.com/kanboard/kanboard/blob/master/app/Model/LinkModel.php

Thanks, sounds legit! A bit above my skill grade, I am afraid.